conf/pacific/rados/13-node-cluster.yaml

```diff
@@ -0,0 +1,73 @@
+# Rados tier-4 test configuration
```

Comment:
Any reason why we are targeting pacific instead of Quincy?

Reply:
no reason. the suite needs to be added to both, and I will add the same in the next PR.

This PR is alrady huge with around ~700 lines of code. Did not want to make it bigger with new suite and conf files for quincy.

Let me know. If necessary to add, will add in the same PR.